api/login.go

```diff
@@ -159,3 +172,38 @@ func GetToken(ctx context.Context, issuerURL, clientID string, usePKCE bool, out
 
 	return nil
 }
+
+type UserInfo struct {
+	Email string
+	Orgs  []string
+}
+
+func GetUserInfoFromToken(tokenString string) (*UserInfo, error) {
```

Comment:
Looks like this will probably break for service accounts. It's already possible to login with a service account token using the flag `nctl auth login --api-token`. It's a bit harder to figure out the org from a service account token as it could live in a project namespace (we'd have to make an API call for sure). But maybe in the meantime we can force the user to supply the org also via a flag when `--api-token` is given.

Reply:
switch the order and added a org parameter when providing an api token